.github/workflows/ci.yml

```
@@ -0,0 +1,90 @@
+      - name: Run tests
+        run: cargo test --workspace --verbose
+        env:
+          RUST_BACKTRACE: 1
+
+  release:
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/v')
+    needs: test
+    steps:
+      - uses: actions/checkout@v4
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)